releases/v2.8.1.toml

```diff
@@ -0,0 +1,25 @@
+# commit to be tagged for new release
+commit = "HEAD"
+
+project_name = "registry"
+github_repo = "distribution/distribution"
+
+# previous release
+previous = "v2.8.0"
+
+pre_release = false
+
+preface = """\
+The 2.8.1 registry release fixes the Go module issues that have popped up in the v2.8.0
```

Comment:
Should we include a mention something like "no changes in binary releases" ?

Reply:
Can do.

Reply:
https://github.com/distribution/distribution/pull/3596/commits/6fa1a7d95171cfe54da5da54ab33c59991501f3c